submodules/power_service/README.md

```diff
@@ -0,0 +1,66 @@
+# SubModule power-service
+
+This submodule is used to create a PowerVS service, SSH Key, and 2 private networks for management network and backup network

```

Comment:
submodule -> module

Reply:
This is the readme for the submodule not the main module

Reply:
Correct, but I think it can be used independently. All submodules are modules themselves, so I would phrase the readme independently from the repo structure